tools/configure/configure.ml

```diff
@@ -22,6 +22,8 @@ let (/) = Filename.concat
 let coq_version = "8.16+alpha"
 let vo_magic = 81591
 let is_a_released_version = false
+let coq_version_hash = fst (run "git" ["rev-parse"; "HEAD"])
```

Comment:
When run from somewhere not a git repo this will fail
It also makes the dune rule unsound

Reply:
What should we do instead?

Reply:
Would putting it in a `try` help?

Reply:
@jfehrle The error is easily avoidable, the differing program behaviour when `git` is or isn't around however is not. It is not clear to me that we can access the git hash in a way that is invariant to the presence of git.